### PR TITLE
Fix issue with TextField and TextArea not handling null

### PR DESCRIPTION
**Pull request by mattpauldavies**
**Backwards Compatibility Implications**

_None_

**New Features**

_None_

**Bug Fixes**

- fix issue with TextField and TextArea not handling null

### CODE REVIEW

File: src/components/Fields/TextArea.story.js [hunk 1]
```diff
@@ -40,6 +40,28 @@ stories.add(
             placeholder="your placeholder here"
             value="Hello, Friend"
         />,
+        <div>
+            <TextArea
+                label="Number Zero"
+                placeholder="your placeholder here"
+                value={0}
```
Comment (jfrej) [inline src/components/Fields/TextArea.story.js]: `TextArea` doesn't accept numbers, so this logs an error to the console.

Comment (mattpauldavies) [reply src/components/Fields/TextArea.story.js]: valid point, I'll remove!

File: src/components/Fields/TextArea.test.js [hunk 2]
```diff
@@ -277,4 +277,24 @@ describe('TextArea', () => {
...
+
+    it('does not add --has-value when value is null', () => {
+        const textField = shallow(<TextArea value={null} />);
+        expect(textField.hasClass('uir-text-area--has-value')).to.equal(false);
+    });
+
+    it('adds --has-value when value is falsy but not null', () => {
+        const textField = shallow(<TextArea value={0} />);
+        expect(textField.hasClass('uir-text-area--has-value')).to.equal(true);
+    });
```
Comment (jfrej) [inline src/components/Fields/TextArea.test.js]: I'm not sure if we need this one if TextArea doesn't accept numbers

Comment (jfrej) [inline src/components/Fields/TextArea.test.js]: I'm not sure if we need this one if TextArea doesn't accept numbers

**Pull request merged**